anubis.go

```diff
@@ -11,6 +13,9 @@ var Version = "devel"
 // access.
 const CookieName = "within.website-x-cmd-anubis-auth"
 
+// CookieExpirationDuration is the amount of time before the cookie/JWT expires.
+const CookieExpirationDuration = 7 * 24 * time.Hour
```

Comment:
Given that this isn't the actual value used. I would remove this and just rely on the flag

Reply:
Or rename to default cookie expiry time